examples/tdec/simple_tdec_local.py

```diff
@@ -0,0 +1,49 @@
+from nucypher.characters.chaotic import NiceGuyEddie, ThisBobAlwaysDecrypts
+from nucypher.policy.conditions.lingo import ConditionLingo
+
+plaintext = b"paz al amanecer"
+THIS_IS_NOT_A_TRINKET = 55
+
+enrico = NiceGuyEddie(encrypting_key=THIS_IS_NOT_A_TRINKET)
+bob = ThisBobAlwaysDecrypts(domain="lynx", eth_provider_uri="Nowhere")
+
+ANYTHING_CAN_BE_PASSED_AS_RITUAL_DATA = 55
+
+before_the_beginning_of_time = {
+    "version": ConditionLingo.VERSION,
+    "condition": {
+        "chain": 1,
+        "method": "blocktime",
+        "returnValueTest": {"comparator": "<", "value": 0},
+    },
+}
+
+ciphertext, tdr = enrico.encrypt_for_dkg_and_produce_decryption_request(
+    plaintext=plaintext,
+    conditions=before_the_beginning_of_time,
+    ritual_id=ANYTHING_CAN_BE_PASSED_AS_RITUAL_DATA,
+)
+
+decrypted_cleartext_from_ciphertext_list = bob.threshold_decrypt(
+    ciphertext=ciphertext,
+    ritual_id=ANYTHING_CAN_BE_PASSED_AS_RITUAL_DATA,
+    conditions=before_the_beginning_of_time,
+)
+
+THESE_CAN_BE_FAKE_FOR_THE_PURPOSES_OF_THIS_DEMO = []
+
+cohort = bob._dkg_insight.fake_ritual.fake_nodes
```

Comment:
To keep inner workings hidden, can this be?

```suggestion
ritual = bob.get_ritual_from_id(ritual_id=ANYTHING_CAN_BE_PASSED_AS_RITUAL_DATA)
cohort = bob.resolve_cohort(ritual=ritual, timeout=<>)
```

Then the inner workings can remain hidden.

You can use `ritual` from above on L39 to get the participant public keys.

Reply:
Yeah that's probably a good idea.

Reply:
Remind me why we need to use `decrypt_using_existing_decryption_request()` and not just `threshold_decrypt()`? Why save the ThresholdDecryptionRequest in this script? Also, if `decrypt_using_existing_decryption_request()` is a function on Bob, could this function just resolve the cohort internally instead of taking it as a parameter?

Reply:
> Why save the ThresholdDecryptionRequest in this script? 

One of the emerging interfaces is a file format, `.tdr`, which allows people to simply pass a file to the network as a request.

Imagine being able to right-click a file and click "Decrypt with Threshold".  Or pass a file to a discord bot and have it decrypt for you.

@KPrasch points out that it makes more sense to divide this into multiple demos.  Which yeah.  But for now, it's useful to just see both ways.

> Also, if decrypt_using_existing_decryption_request() is a function on Bob, could this function just resolve the cohort internally instead of taking it as a parameter?

Dude heck yeah.  I mean, better even then that is to just include the cohort in the tdr.